src/Http/Http.Features/src/IHttpResponseBodyFeature.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    /// <summary>
+    /// An aggregate of the different ways to interact with the response body.
+    /// </summary>
+    public interface IHttpResponseBodyFeature
+    {
+        /// <summary>
+        /// The <see cref="Stream"/> for writing the response body.
+        /// </summary>
+        Stream Body { get; }
```

Comment:
Should be called Stream

Reply:
I'd be OK with either BodyStream/BodyWriter or just Stream/Writer.